src/snabbdom.ts

```diff
@@ -57,10 +57,29 @@ export function init(modules: Array<Partial<Module>>, domApi?: DOMAPI) {
     }
   }
 
-  function emptyNodeAt(elm: Element) {
+  function toVNode(elm: Element): VNode {
```

Comment:
Why not make this a bit more generic and accept `Node` instead of `Element`? This would make it possible to patch a virtual node against a `Text` node (and since a recent PR, against `Comment` nodes too).